pkgs/by-name/pd/pdfid/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, fetchzip
+, python3
+, python3Packages
+, makeWrapper
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "pdfid";
+  version = "0.2.8";
+  format = "other";
+
+  src = fetchzip {
+    url = "https://didierstevens.com/files/software/pdfid_v0_2_8.zip";
+    hash = "sha256-ZLyhBMF2KMX0c1oCvuSCjEjHTnm2gFhJtasaTD9Q1BI=";
+    stripRoot=false;
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
```

Comment:
I would use `makeBinaryWrapper` instead, it has a better compatibility with Darwin platforms.